fastembed/common/preprocessor_utils.py

```diff
@@ -68,7 +68,19 @@ def load_tokenizer(model_dir: Path) -> Tuple[Tokenizer, dict]:
             token_str = token.get("content", "")
             special_token_to_id[token_str] = tokenizer.token_to_id(token_str)
 
-    return tokenizer, special_token_to_id
+    if tokenizer_config["tokenizer_class"] == "BertTokenizer":
+        query_marker = {"[Q]": 1}
+        document_marker = {"[D]": 2}
+    elif tokenizer_config["tokenizer_class"] == "XLMRobertaTokenizer":
+        query_marker = {"[QueryMarker]": 250002}
+        document_marker = {"[DocumentMarker]": 250003}
+    else:
+        query_marker = {}
+        document_marker = {}
```

Comment:
it should be specified in the particular implementations of models' classes which require this